src/entity/aggregator.rs

```diff
@@ -46,6 +48,15 @@ impl Model {
     pub fn is_tombstoned(&self) -> bool {
         self.deleted_at.is_some()
     }
+
+    pub fn is_first_party(&self) -> bool {
+        // probably temporary
+        matches!(self.dap_url.domain(), Some(domain) if domain.ends_with("divviup.org"))
```

Comment:
note that this temporary fix would allow someone to sign up for malicious-divviup.org and create an aggregator that we treat as first party, which I don't think is a particularly big concern but isn't a great long term choice

Reply:
This gets fixed by #217, right? If so let's leave a reference to the issue in a comment here.